dist/angular-validation.js

```diff
@@ -270,6 +270,28 @@
     };
 
     /**
+     * Add Message Element at the initial phase
+     * Do this function when to custom your messageElement 
+     * NODE: this funtion have similar purpose with `message-id` attribute. 
+     * This function will help you add your `messageElement` automatically instead of pre-defined.
+     * @param element
+     */
+    this.addMsgElement = function(element) {
+      return element.after('<span></span>');
```

Comment:
Also, I would like to use `set` instead of `add`, as you can see other providers function name, they are all `set` and `get` to name it.